src/components/layout.js

```diff
@@ -45,71 +55,53 @@ export default class Layout extends React.PureComponent {
     },
   ]
 
-  footerLinks = [
+  const footerLinks = [
     {
       name: "Join Us",
-      endpoint: "/",
+      anchor: true,
+      endpoint: "https://www.codeforsanjose.com/",
     },
     {
       name: "Find your ballot",
       endpoint: "/findYourBallot",
     },
     {
       name: "Open source",
-      endpoint: "/",
+      anchor: true,
+      endpoint: "https://github.com/codeforsanjose/open-disclosure",
     },
     {
-      name: "Press",
-      endpoint: "/",
+      name: "Register to vote",
+      endpoint: "/registerToVote",
     },
   ]
 
-  handleSubmit() {}
+  return (
+    <div className={styles.container}>
+      <Navbar links={links} windowIsLarge={props.windowIsLarge} />
 
-  handleChange = ({ target: { value } }) => {
-    this.setState({ value })
-  }
+      {props.children}
 
-  renderNavItems = items => (
-    <ul>
-      {items.map((item, index) => (
-        <li
-          key={`footer nav item ${item.name}`}
-          className={styles.footerNavItem}
-        >
-          <Link to={item.endpoint}>{item.name}</Link>
-        </li>
-      ))}
-    </ul>
-  )
-
-  render() {
-    return (
-      <div className={styles.container}>
-        <Navbar links={this.links} windowIsLarge={this.props.windowIsLarge} />
-
-        {this.props.children}
-
-        <footer className={styles.footer}>
-          <div className={styles.footerInner}>
-            <div className={styles.footerTop}>
-              <div className={styles.footerLeft}>
-                <Logo />
-                <nav className={styles.footerNav}>
-                  {this.renderNavItems(this.links)}
-                  {this.renderNavItems(this.footerLinks)}
-                </nav>
-              </div>
-              <div className={styles.footerRight}>
+      <footer className={styles.footer}>
+        <div className={styles.footerInner}>
+          <div className={styles.footerTop}>
+            <div className={styles.footerLeft}>
+              <Logo />
+              <nav className={styles.footerNav}>
+                {renderNavItems(links)}
+                {renderNavItems(footerLinks)}
+              </nav>
+            </div>
+            {/* <div className={styles.footerRight}>
```

Comment:
Did you mean to keep this commented out section?

Reply:
Yes; that's a functionality we don't support yet but plan to in the future. It's just been completely styled so I don't want to get rid of it.